src/Campaigns/Actions/AddNewBadgeToAdminMenuItem.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+/**
+ * @unreleased
+ */
+class AddNewBadgeToAdminMenuItem {
+    /**
+     * @return void
+     */
+    public function __invoke()
+    {
+        // only show bade for existing users who have upgraded from a version prior to 4.0.0
+        if (version_compare((string)get_option('give_version_upgraded_from', ''), '4.0.0', '>=')) {
+            return;
+        }
```

Comment:
Could we fix the typo in _bade_? Also, shouldn’t we set the default value of the option to `4.0.0`? Comparing an empty string against version `4.0.0` will return `false`, causing the badge to be shown, which is the expected behavior in a fresh install.

Reply:
Good catch! Resolved 1ad4d7cc